x-pack/test/common/services/security_solution/roles_users_utils.ts

```diff
@@ -88,53 +44,9 @@ export const createUserAndRole = async (
  */
 export const deleteUserAndRole = async (
   getService: FtrProviderContext['getService'],
```

Comment:
nit: not related to changes in the PR but the JSDOC for the function needs an update